ProcessLib/TES/TESOGS5MaterialModels.h

```diff
@@ -0,0 +1,399 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#ifndef PROCESSLIB_TES_OGS5MATERIALMODELS
+#define PROCESSLIB_TES_OGS5MATERIALMODELS
+
+#include "TESAssemblyParams.h"
+
+namespace
+{
+const double GAS_CONST = 8.3144621;
+}
+
+namespace ProcessLib
+{
+namespace TES
+{
+
+inline double fluid_density(const double p, const double T, const double x)
+{
+	// OGS-5 density model 26
```

Comment:
Shall I put a reference to a specific commit on GitHub? :smile:
Please don't complain too much about this file. It's mostly copied from OGS5, hence the name. Again: I'd like to do documentation within a general cleanup.